src/index.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash';// eslint-disable-line no-unused-vars
+import './style.css';
+
+const arrTasks = [
+  {
+    description: 'forth',
+    completed: false,
+    index: 3,
+  },
+  {
+    description: 'first',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'thirth',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'second',
+    completed: false,
+    index: 1,
+  },
```

Comment:
✅ It has an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\] 👍🏻